rust/docker-init.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+if [ $ENABLE_MASQUERADE = "1" ]; then
+  IFACE="utun"
+  iptables -A FORWARD -i $IFACE -j ACCEPT; iptables -A FORWARD -o $IFACE -j ACCEPT; iptables -t nat -A POSTROUTING -o eth+ -j MASQUERADE
```

Comment:
I guess there is no need to keep it as an one-liner
```suggestion
  iptables -A FORWARD -i $IFACE -j ACCEPT
  iptables -A FORWARD -o $IFACE -j ACCEPT
  iptables -t nat -A POSTROUTING -o eth+ -j MASQUERADE
```

Reply:
fixed in #1729